src/index.html

```diff
@@ -382,7 +382,7 @@ <h1 class="c-section__header js-lang" data-key="cta-h1">Zapraszamy <br class="d-
                                 <ul class="c-list">
                                     <p class="c-list__title js-lang" data-key="map-social">Aurator w social media:</p>
                                     <li class="c-list__item">
-                                        <a href="https://www.facebook.com/Aurator-Sp-z-oo-1606132342787317" target="_blank"><img class="c-contact__icon" src="img/c-contact__icon--fb.png" alt="facebook"></a>
+                                        <a class="h-orange" href="https://www.facebook.com/Aurator-Sp-z-oo-1606132342787317" target="_blank">facebook</a>
```

Comment:
- [ ] Please remove image file

Reply:
Let's do this kind of cleaning up, after we get final acceptance from Aurator. Just in case...